models/eos/ntp_global/eos_ntp_global.yml

```diff
@@ -0,0 +1,152 @@
+notes:
+- Tested against Arista EOS 4.24.60M
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+options:
+    config:
+      description: A dictionary of ntp options
+      type: dict
+      suboptions:
+        authenticate:
+          description:
+          - Require authentication for NTP synchronization.
+          type: dict
+          suboptions:
+            enable:
+              description: Enable authentication for NTP synchronization.
+              type: bool
+            servers:
+              description: Authentication required only for incoming NTP server responses.
+              type: bool
+        authentication_key:
+          description:
+          - Define a key to use for authentication.
+          type: list
+          elements: dict
+          suboptions:
+            id:
+              description: key identifier.
+              type: int
+            algorithm:
+              description: hash algorithm,
+              type: str
+              choices: ["md5", "sha1"]
+            encryption:
+              description: key type
+              type: int
+              choices: [0, 1]
+            key:
+              description: Unobfuscated key string.
+              type: str
+              no_log: True
+        local_interface:
+          description: Configure the interface from which the IP source address is taken.
+          type: str
+        qos_dscp:
+          description: Set DSCP value in IP header
+          type: int
+        serve:
+          description: Configure the switch as an NTP server.
+          type: dict
+          suboptions:
+            all:
+              description: Service NTP requests received on any interface.
+              type: bool
+            access_lists:
+              description: Configure access control list.
+              type: list
+              elements: dict
+              suboptions:
+                afi:
+                  description: ip/ipv6 config commands.
+                  type: str
+                acls:
+                  description: Access lists to be configured under the afi
+                  type: list
+                  elements: dict
+                  suboptions:
+                    acl_name:
+                      description: Name of the access list.
+                      type: str
+                    direction:
+                      description: direction for the packets.
+                      type: str
+                      choices: ["in", "out"]
+                    vrf:
+                      description: VRF in which to apply the access control list.
+                      type: str
+        servers:
+          description: Configure NTP server to synchronize to.
+          type: list
+          elements: dict
+          suboptions:
+            vrf:
+              description: vrf name.
+              type: str
+            host:
```

Comment:
For consistency with how we have historically dealt with these attributes (neighbors -> neighbor), can we call it server?
```suggestion
            server:
```